vendor/engines/split_accounts/app/views/facility_accounts/account_fields/split_accounts/_splits.html.haml

```diff
@@ -0,0 +1,17 @@
+- available_subaccounts = Account.excluding_split_accounts.filter_global_account_types
+= javascript_include_tag "split_account_splits"
+
+%h4 Subaccounts
+
+%div{data: {subaccounts: true}}
+  = f.nested_fields_for :splits, class_name: "SplitAccounts::Split" do |ff|
+    .well
+      = ff.input :subaccount_id, collection: available_subaccounts, wrapper_html: {class: "input-inline"}
```

Comment:
One worry I have here is that the list will be _very_ long in NU. It might be fine if we just wrap it in Chosen, but we might want to consider an ajax lookup. We can probably leave this as is for the current PR, but we'll probably want to do something at a later point.